.circleci/config.yml

```diff
@@ -183,11 +395,11 @@ workflows:
               only:
                 - develop
     jobs:
-      - osteo:
+      - osteo-build-and-deploy:
           name: osteo-nightly
-      - angio:
+      - angio-build-and-deploy:
           name: angio-nightly
-      - brain:
+      - brain-build-and-deploy:
           name: brain-nightly
-      - mbc:
+      - mbc-build-and-deploy:
           name: mbc-nightly
```

Comment:
I left the nightly builds, but I suspect we don't need them. Originally idea was to do --aot builds only at night to make them quicker (I think). Reality is we only save a couple of minutes and saves us pain to find out next day that aot broke the build.